tests/Klasse.spec.ts

```diff
@@ -133,4 +133,52 @@ test.describe(`Testfälle für die Administration von Klassen: Umgebung: ${proce
       await expect(klasseCreationView.button_ZurueckErgebnisliste).toBeVisible();
     });
   });
+
+  test("Jede Klasse hat eine Dienststellennummer neben dem Klassennamen (mit Paginierung)", { tag: [LONG, SHORT, STAGE] }, async ({ page }) => {
+    const startseite = new StartPage(page);
+    const menue = new MenuPage(page);
+    const klasseManagementView = new KlasseManagementViewPage(page);
+
+    await test.step(`Klassenverwaltung öffnen und prüfen, dass jede Klasse eine Dienststellennummer hat`, async () => {
+        // Navigate to Klassenverwaltung
+        await startseite.card_item_schulportal_administration.click();
+        await menue.menueItem_AlleKlassenAnzeigen.click();
+
+        // Wait until the table is visible
+        await expect(klasseManagementView.text_h2_Klassenverwaltung).toHaveText("Klassenverwaltung");
+
+        // Define the selector for the 'Next' button
+        const nextPageButton = page.locator('.v-pagination__next button:not(.v-btn--disabled)');
+
+        // Loop over each page
+        let hasNextPage = true;
+        while (hasNextPage) {
+            // Get all rows in the current page's Klasse table
+            const rows = page.locator('table >> tbody >> tr');
+            const rowCount = await rows.count();
+
+            // Iterate over each row to check if Dienststellennummer and Klassenname are visible and not empty
+            for (let i = 0; i < rowCount; i++) {
```

Comment:
Maybe you can create a iteration and compare function in the page file KlasseManagementView.page.ts?
Something like that:
 public async iterateDataTable(xx) {
    xxx
  }

Reply:
Thought maybe it's not good practice to "expect" in a page. It's done.